database.go

```diff
@@ -184,7 +185,7 @@ func (db *BadgerDB) ClearAll() error {
 }
 
 // Subscribe to watch for changes for the given prefixes
-func (db *BadgerDB) Subscribe(ctx context.Context, cb func(kv *KVList) error, prefixes []byte) error {
+func (db *BadgerDB) Subscribe(ctx context.Context, cb func(kv *KVList) error, prefixes []pb.Match) error {
```

Comment:
what is `pb.Match`?

Reply:
the type `pb.Match` which is a structure defined here: https://github.com/dgraph-io/badger/blob/31e5a1bd0e04a5a0583169d5298b61744f105996/pb/badgerpb3.pb.go#L501, and it holds a slice of bytes, so the main change is that from an unwrapped byte slice to a structure that holds it. This type was introduced by `badger/v3`